devel/meson/src/openwrt-cross.txt.in

```diff
@@ -4,22 +4,22 @@ cpp = [@CXX@]
 ar = '@AR@'
 strip = '@STRIP@'
 nm = '@NM@'
-ld = '@LD@'
 pkgconfig = '@PKGCONFIG@'
 
-[properties]
+[built-in options]
 c_args = [@CFLAGS@]
 c_link_args = [@LDFLAGS@]
 cpp_args = [@CXXFLAGS@]
 cpp_link_args = [@LDFLAGS@]
-needs_exe_wrapper = true
 
 [host_machine]
 system = 'linux'
 cpu_family = '@ARCH@'
 cpu = '@CPU@'
 endian = '@ENDIAN@'
 
+[properties]
+needs_exe_wrapper = true
+
 [paths]
 prefix = '/usr'
-libdir = 'lib'
```

Comment:
Did that error out or give a warning?
IIRC I added that for debian/multilib because sometimes strings a la `x86_64-linux-gnu` and such could end up in there?

Reply:
I haven't seen that. meson docs say that for cross compilation, libdir defaults to lib. for the native file it absolutely is needed.